Assets/Mirror/Examples/AdditiveLevels/Scripts/Portal.cs

```diff
@@ -69,7 +69,7 @@ IEnumerator SendPlayerToNewScene(GameObject player)
                 yield return new WaitForSeconds(AdditiveLevelsNetworkManager.singleton.fadeInOut.GetDuration());
 
                 // Remove player after fader has completed
-                NetworkServer.RemovePlayerForConnection(conn, false);
+                NetworkServer.RemovePlayerForConnection(conn, NetworkServer.RemovePlayerOptions.Unspawn);
```

Comment:
Probably easier to have RemovePlayerOptions directly accessible instead of under NetworkServer.* namespace.
This looks a little verbose

Reply:
Done